Donation-platform/src/main/java/com/google/DbSetUpUtils.java

```diff
@@ -38,6 +38,13 @@
   private static final String RACE_EQ = "racial equality";
   private static final String HEALTH = "health";
 
+  private static final String HUNGER_IMAGE = "https://thumbs.dreamstime.com/b/eating-utensils-icon-trendy-eating-utensils-logo-concept-whi-eating-utensils-icon-trendy-eating-utensils-logo-concept-white-131149320.jpg";
```

Comment:
i hope Ryan is also using the same image for tags because i saw him also adding some icons

Reply:
please co-ordinate with him

Reply:
yes we're gonna work on which images to show, these are just placeholders

Reply:
Yep, we'll coordinate. Currently I'm using icons from font awesome, which is included in the MDB imports.